contributing/DOCS-CONTRIBUTING.md

```diff
@@ -68,45 +71,203 @@ documentation efforts, who are happy to help you get started.
 
 There are a couple different ways to jump in to the Knative doc set:
 
-- Look at issues in the Docs repo labled
-  [Good First Issue](https://github.com/knative/docs/labels/kind%2Fgood-first-issue).
-- Run through the [install guide](../install/README.md) for the platform of your
-  choice, as well as the
-  [Getting Started with Knative App Deployment](../install/getting-started-knative-app.md)
-  guide, and keep a
-  [friction log](https://devrel.net/developer-experience/an-introduction-to-friction-logging)
-  of the experience. What was hard for you? Then open a PR with a few
-  improvements you could make from the friction log you kept. How can you make
-  the documentation better so that others don't run into the same issues you
-  did?
+- Look for a
+  [Good First Issue](https://github.com/knative/docs/labels/kind%2Fgood-first-issue)
+  in the backlog.
 
-### Submitting documentation PRs - what to expect
+- Try out Knative and send us feedback. For example, run through one of the
+  [install guides](../install/README.md) and then try
+  [Getting Started with Knative Serving](../install/getting-started-knative-app.md).
+
+    You should keep a
+    [friction log](https://devrel.net/developer-experience/an-introduction-to-friction-logging)
+    of your experience and then use that to open your first set of PRs.
+    Examples:
+
+   - What was hard for you?
+   - Did you stumble on something because it wasn't clear?
+   - Was a dependency not mentioned?
+
+**A few pointers and other considerations**
+
+  - Think about how you can improve the documentation as a whole. <br>
```

Comment:
```suggestion
  - Think about how you can improve the documentation as a whole.
```

Reply:
the spacing is wrong in the website, thus the manual line break. this does not render in github nor the website. so leaving this in unless you feel strongly about removing it

Reply:
huh. I'd like to not have random bits of HTML in the docs. Can we take it out and then open an issue in the website repo about the rendering problem, if it persists? Maybe it was a quirk that'll go away?

Reply:
no quirk its a markdown renderer thing that wont go away unless we use all hmtl. it should be obvious to those familiar with markdown and html. an issue against the website repo wont fix the weird whitespace or lack of it